server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader11to12.java

```diff
@@ -209,4 +216,16 @@ static void upgradeDataFileCF(final Key key, final Value value, final Mutation m
     }
   }
 
+  public void removeScanServerRange(ServerContext context, String tableName) {
```

Comment:
The upgrade code will also need to create the new table.  In #4690 it creates the new table in the init code.  Should try to reuse code for that.

Reply:
Found [this](https://github.com/apache/accumulo/blob/47a53ae91d223021155e05a71f492729a56c012d/server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java#L81) in the elasticity upgrade code to create a fate table.  But not sure if that is doing the same thing init does when it creates the fate table.

Reply:
We can leave #4652 open if the table creation is not done.  Table creation in upgrade does not need to block this PR, we just need to ensure its not forgotten.

Reply:
Was looking into what the upgrade code in elasticity does for upgrade and it does not seem complete.  Opened #4692 and commented on #4652 after looking into this.

Reply:
Added the table creation bits to this for zk table node creation, populating of table properties, and initial tablet file creation.

Reply:
In order to reuse the init code I had to make the class public. 

I'm not a huge fan of that as a solution so I'm open to suggestions on where this table creation code should live.